orderer/common/follower/follower_chain.go

```diff
@@ -172,7 +172,7 @@ func NewChain(
 
 		// Check the block puller creation function once before we start the follower. This ensures we can extract
 		// the endpoints from the join-block.
-		puller, err := blockPullerFactory.BlockPuller(joinBlock)
+		puller, err := blockPullerFactory.BlockPuller(joinBlock, make(chan struct{}))
```

Comment:
Nit: why not just pass nil here if we don't expect to use the channel?

Reply:
Done